internal/pod_models_test.go

```diff
@@ -560,7 +560,7 @@ var _ = Describe("pod_models", func() {
 				It("should have the sidecar container", func() {
 					sidecarContainer := spec.Containers[1]
 					Expect(sidecarContainer.Name).To(Equal(fdbv1beta2.SidecarContainerName))
-					Expect(sidecarContainer.Image).To(Equal(fmt.Sprintf("foundationdb/foundationdb-kubernetes:%s", cluster.Spec.Version)))
+					Expect(sidecarContainer.Image).To(And(HavePrefix(fdbv1beta2.FoundationDBKubernetesBaseImage), HaveSuffix(cluster.Spec.Version)))
```

Comment:
wouldn't this be true for all of the BaseImage variables?

Reply:
no -- I was looking at the prior variable but `FoundationDBKubernetesBaseImage = "foundationdb/fdb-kubernetes-monitor"` is not a prefix for the others like `"foundationdb/foundationdb-kubernetes"` would be